### PR TITLE
Staking form added feature and bug fixes

### PR DESCRIPTION
**Pull request by nqsullivan**
- Fixed 'MAX AMOUNT' button for differing window sizes 29a2af2fa0849f4d9c366d9008efc619cdbaa915
- Changed helper text in staking form to more accurately reflect error 599af6d4d6ed7a1af169197446d536fc6e635ed1
- Changed label in staking form to a less verbose form with mobile users in mind 0ac2d0f863fd3380921d48c3b962ffd1ab32eb06
- Added a more intuitive way for a user to exit the form 3eb22781b2d8b84de6a3ca52d7553cc54222e62a
- Updated theme to include colors for cancel button 45ea05d7f24a941a8920d147840419cb5ccf8028

### CODE REVIEW

File: pages/staking.jsx [hunk 1]
```diff
@@ -587,46 +587,39 @@ const Staking = () => {
...
                 )}
-                <Grid
-                  container
-                  spacing={3}
-                  alignItems="stretch"
-                  justifyContent="center"
-                  sx={{ flexGrow: 1 }}
-                >
-                  <Grid item md={10} xs={9}>
+                <Grid container sx={{mb: 2}} justifyContent={'space-between'} alignItems={'flex-start'}>
```
Comment (noob77777) [inline pages/staking.jsx:590]: Minor: Code formatting
```suggestion
                <Grid
                    container
                    sx={{ mb: 2 }}
                    justifyContent="space-between"
                    alignItems="flex-start"
                >
```

File: pages/staking.jsx [hunk 2]
```diff
@@ -587,46 +587,39 @@ const Staking = () => {
...
+                  <Grid item md={9} xs={9} sx={{ minWidth: 0}}>
```
Comment (noob77777) [inline pages/staking.jsx:591]:
```suggestion
                  <Grid item md={9} xs={9} sx={{ minWidth: 0 }}>
```

**Review by noob77777 — CHANGES_REQUESTED** [review]

**Comment by noob77777** [comment]
PRs should be made for ```dev``` branch and not directly to ```main```

File: pages/staking.jsx [hunk 3]
```diff
@@ -587,46 +587,39 @@ const Staking = () => {
...
                       name="stakingAmount"
                       variant="filled"
-                      sx={{ mb: 2 }}
                       onChange={handleStakingFormChange}
                       value={stakingForm.tokenAmount}
                       error={stakingFormErrors.tokenAmount}
                       helperText={
-                        stakingFormErrors.tokenAmount &&
-                        'Enter a valid token amount'
+                        stakingFormErrors.tokenAmount && (tokenBalance ? 'Enter a valid token amount' : 'Please connect with yoroi or nautilus to use MaxAmount')
```
Comment (nqsullivan) [inline pages/staking.jsx:609]:
```suggestion
                        stakingFormErrors.tokenAmount && (dAppWallet.connected ? 'Enter a valid token amount' : 'Please connect with yoroi or nautilus to use MaxAmount')
```

File: pages/staking.jsx [hunk 4]
```diff
@@ -587,46 +587,39 @@ const Staking = () => {
...
-                            name: 'stakingAmount',
-                            value: tokenBalance,
-                          },
-                        });
-                      }}
-                    >
-                      Max Amount
-                    </Button>
-                  </Grid>
+                    <Grid item md={2} xs={2} sx={{ mt: 0.5, minWidth: '63.03px', maxWidth: '63.03px'}}>
```
Comment (nqsullivan) [inline pages/staking.jsx:613]: Changed column count and changed width value to rem to allow for padding per comment
```suggestion
                    <Grid item md={2} xs={3} sx={{minWidth: '4.5rem', maxWidth: '4.5rem'}}>
```

File: pages/staking.jsx [hunk 5]
```diff
@@ -587,46 +587,39 @@ const Staking = () => {
...
+                      <Button
+                          sx={{p: 0,  minWidth: '63.03px', maxWidth: '63.03px'}}
```
[the remark this answers is not in the document]
Comment (nqsullivan) [reply pages/staking.jsx:615]: Changed column count and changed width value to rem to allow for padding per comment
```suggestion
                          sx={{p: '0.25rem',  minWidth: '4.5rem', maxWidth: '4.5rem'}}
```

File: pages/staking.jsx [hunk 6]
```diff
@@ -650,7 +643,7 @@ const Staking = () => {
                     type="ergoAddress"
                     sx={{
                       width: '100%',
-                      border: '1px solid rgba(82,82,90,1)',
+                      border: '1px solid #F53F3F',
```
[the remark this answers is not in the document]
Comment (nqsullivan) [reply pages/staking.jsx:651]:
```suggestion
                      border: '1px solid rgba(82,82,90,1)',
```

**Review by nqsullivan — COMMENTED** [review]
Addressing review

File: pages/staking.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (nqsullivan) [reply pages/staking.jsx:609]:
```suggestion
                        stakingFormErrors.tokenAmount && (dAppWallet.connected ? 'Enter a valid token amount' : 'Please connect with yoroi or nautilus to use MaxAmount')
```

File: pages/staking.jsx [hunk 7]
```diff
@@ -688,6 +681,31 @@ const Staking = () => {
...
+                        background: theme.palette.quinary.active,
+                      },
+                    }}
+                    onClick={() => {
+                      setOpenModal(false);
+                      setTransactionSubmitted(null);
+                    }}
+                >
+                  Cancel
+                </Button>
```
[the remark this answers is not in the document]
Comment (noob77777) [reply pages/staking.jsx:713]: yes probably a better approach. Also no need to add a 5th color to theme.js

File: styles/theme.js [hunk 8]
```diff
@@ -35,6 +35,11 @@ let theme = createTheme({
     quaternary: {
       main: 'rgb(237, 100, 166)',
     },
+    quinary: {
+      main: '#F53F3F',
+      hover: '#F36666',
+      active: '#F71818'
+    },
```
Comment (noob77777) [inline styles/theme.js:42]: not required.

**Review by noob77777 — CHANGES_REQUESTED** [review]

File: pages/staking.jsx [hunk 9]
```diff
@@ -588,45 +588,43 @@ const Staking = () => {
...
-                            name: 'stakingAmount',
-                            value: tokenBalance,
-                          },
-                        });
-                      }}
-                    >
-                      Max Amount
-                    </Button>
-                  </Grid>
+                    <Grid item md={2} xs={2} sx={{ mt: 0.5, minWidth: '63.03px', maxWidth: '63.03px'}}>
```
Comment (nqsullivan) [inline pages/staking.jsx]:
```suggestion
                  <Grid item md={2} xs={3} sx={{ minWidth: '4.5rem', maxWidth: '4.5rem'}}>
```

File: pages/staking.jsx (hunk 7 above)
[the remark this answers is not in the document]
Comment (nqsullivan) [reply pages/staking.jsx:713]: Ok I will recreate this pr to the dev branch and include a button like the one in the connect wallet modal

**Pull request closed without merging**